src/components/Filter/Filter.js

```diff
@@ -0,0 +1,314 @@
+                    value={payRange[1]}
+                  />
+                </BodyWrapperInner>
+              </BodyWrapper>
+            </BodyInner>
+          </BodyContainer>
+        </BodyDiv>
+        <BodyDiv>
+          <H2>공간 유형</H2>
+          {ROOM_TYPE.map(type => {
+            return (
+              <BodyTypeDiv key={type.id}>
+                <BodyTypeFilter>
+                  <CheckboxInput
+                    type="checkbox"
+                    name={type.name}
+                    id={type.id}
+                    checked={checkedList.includes(type.name) ? true : false}
+                    onClick={e => {
+                      handleChecked(e);
+                    }}
+                  />
+                  <BodyTypeFilterInner>
+                    <TitleSpan>{type.title}</TitleSpan>
+                    <GraySmallSpan>{type.explanation}</GraySmallSpan>
+                  </BodyTypeFilterInner>
+                </BodyTypeFilter>
+              </BodyTypeDiv>
+            );
+          })}
+        </BodyDiv>
+        <BodyDiv>
+          <H2>편의시설</H2>
+          {ROOM_PROVIDE.map(type => {
+            return (
+              <BodyTypeDiv key={type.id}>
+                <BodyTypeFilter>
+                  <CheckboxInput
+                    type="checkbox"
+                    name={type.name}
+                    id={type.id}
+                    checked={checkedList.includes(type.name) ? true : false}
+                    onClick={e => {
+                      handleChecked(e);
+                    }}
+                  />
+                  <BodyTypeFilterInner>
+                    <TitleSpan>{type.title}</TitleSpan>
+                  </BodyTypeFilterInner>
+                </BodyTypeFilter>
+              </BodyTypeDiv>
+            );
+          })}
+          {isAdd &&
+            ROOM_PROVIDE_HIDE.map(type => {
+              return (
+                <BodyTypeDiv key={type.id}>
+                  <BodyTypeFilter>
+                    <CheckboxInput
+                      type="checkbox"
+                      name={type.name}
+                      checked={checkedList.includes(type.name) ? true : false}
+                      onClick={e => {
+                        handleChecked(e, e.target.name, type.id);
+                      }}
+                    />
+                    <BodyTypeFilterInner>
+                      <TitleSpan>{type.title}</TitleSpan>
+                    </BodyTypeFilterInner>
+                  </BodyTypeFilter>
+                </BodyTypeDiv>
+              );
+            })}
+          <MoreDiv>
+            <MoreSpan
+              onClick={() => {
+                setIsAdd(!isAdd);
+              }}
+            >
+              {isAdd ? '접기' : '더 보기'}
+            </MoreSpan>
+          </MoreDiv>
+        </BodyDiv>
+      </Body>
+      <Footer>
+        <FooterDiv>
+          <AllCancelDiv>
+            <AllCancelSpan onClick={() => handleCheckCancel()}>
```

Comment:
```suggestion
            <AllCancelSpan onClick={handleCheckCancel}>
```
넘겨줄 인자가 없다면 위와 같이 작성해도 되겠습니다.